mimic_2_data_analysis.ipynb

```diff
@@ -0,0 +1,3433 @@
+{
```

Comment:
<p>Not sure whether this is an issue of reviewnb or so, but this renders weirdly. Every mathematical symbol is on a new line? Think you can ignore my comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>

Reply:
<p>This is consistant to the format in the book, in which every mathematical symbol is on a new line. </p>